### DISCUSSION TITLE
Migrate command succeeds with invalid migration path

### DISCUSSION THREAD

**Discussion by nazmulpcc** [laravel/framework#39607] [Ideas]
- Laravel Version:  8.64.0
- PHP Version: Any
- Database Driver & Version: Any

### Description:
Currently, if you run `php artisan migrate --path=foobar`, the command succeeds even when the given migration path doesn't exist and says 'Nothing to migrate.' in a green font.
This behavior can be confusing and misleading at times. For example this caused me to loose my head when my initialization script ran smoothly but I was missing some tables in my DB. It took me some time to find out that I supplied the wrong path to the migration command. This should throw an error and the command should fail.

### Steps To Reproduce:
Just run `php artisan migrate --path=foobar` in any Laravel project.

**Comment by driesvints** [comment]
Heya, thanks for submitting this.

This seems like a feature request or an improvement so I'm converting this to a discussion instead. It's best to post these in the discussions section in the future to get support for your idea. After that you may send a PR to the framework. Please only use the `laravel/framework` issue tracker to report bugs and issues with the framework.

Thanks!
